Sources/DocCDocumentation/DocCDocumentation.docc/V5_API_Migration_guide.md

```diff
@@ -2,6 +2,8 @@
 
 ## StoreKit 2
 
+> Warning: When upgrading to v5, you **must** configure your [In-App Purchase Key](/service-credentials/itunesconnect-app-specific-shared-secret/in-app-purchase-key-configuration) in the RevenueCat dashboard. **Purchases will fail if the key is not configured**.
+
```

Comment:
Dumb idea: have this as the only text you can read with a button below it saying "I've read the above, please show the migration guide".